### ISSUE TITLE
figcaption fails validation

### ISSUE THREAD

**Issue by moderndeveloperllc** [structured-data/linter#31]
The [figcaption](http://www.w3.org/TR/html5/grouping-content.html#the-figcaption-element) element is being tagged as invalid: `validation https://www.allcarepharmacy.com: Tag figcaption invalid`. Example code:
```
<figcaption itemprop="caption">
    <span itemprop="copyrightHolder" itemscope itemtype="http://schema.org/Organization"> Copyright: <a href="http://www.123rf.com/profile_stylephotographs" itemprop="url">stylephotographs / 123RF Stock Photo</a>
    </span>
</figcaption>
```

**Comment by gkellogg** [comment]
I'll add `figcaption` to the list of ignored errors in the RDFa parser. The problem fundamentally is that Nokogiri incorrectly tags these.

**Comment by gkellogg** [comment]
Should be fixed in 2.3.0.
